### DISCUSSION TITLE
admincyber

### DISCUSSION THREAD

**Discussion by AdminCyber1** [community/community#140219] [Codespaces]
### Select Topic Area

General

### Body

Ingin belajar mengembangkan website

**Comment by 7x9z** [comment]
Hello, @AdminCyber1,
You can learn stuff like this on other websites such as these:
https://learnhtmlcss.online/
https://www.codecademy.com/catalog/language/html-css
https://www.w3schools.com/html/

**Comment by Akash1134** [comment]
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
